cellprofiler/gui/pipelinelistview.py

```diff
@@ -1526,7 +1529,8 @@ def on_paint(self, event):
                         not item.selected):
                         flags |= wx.CONTROL_SELECTED
                 draw_item_selection_rect(self, dc, r, flags)
-                if flags & wx.CONTROL_SELECTED:
```

Comment:
Use the select color (usually white) only if selected and focused (on a dark blue background). Otherwise, use black.